core/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/JsonCodec.scala

```diff
@@ -194,8 +198,8 @@ object JsonCodec {
   implicit val networkFieldsEncoder: Encoder[NetworkFields] =
     Encoder.forProduct3("networkName", "subNetworkName", "subNetworkIpRange")(x => NetworkFields.unapply(x).get)
   implicit val kubeAsyncFieldEncoder: Encoder[KubernetesClusterAsyncFields] =
-    Encoder.forProduct3("loadBalancerIp", "apiServerIp", "networkInfo")(x =>
-      KubernetesClusterAsyncFields.unapply(x).get
+    Encoder.forProduct3("loadBalancerIp", "apiServerIp", "networkInfo")(
```

Comment:
are these from scalafmt?

Reply:
I thought I'd run scalafmt but I hadn't. Thanks for pointing it out!